### ISSUE TITLE
Noisy logging

### ISSUE THREAD

**Issue by larsrh** [sbt/sbt-pgp#97]
I recently bumped sbt-pgp to 1.0.1 and found this strange log entries in my output:

```
background log: info: Running info.hupel.isabelle.cli.Main --version 2016 --session Protocol --internal build
background log: debug: Waiting for threads to exit or System.exit to be called.
background log: debug:   Classpath:
...
``` 

Some debugging leads me to believe that this is caused by [sbt-core-next](https://github.com/sbt/sbt-core-next).

There's a workaround mentioned in sbt/sbt-core-next#8, but I wonder whether this should be fixed once and for all.

**Comment by xerial** [comment]
I've also seen this annoying log message. It seems some library or sbt-pgp is setting java.util.logging log level.

**Comment by xerial** [comment]
OK. The cause is https://github.com/sbt/sbt-core-next/issues/8

**Comment by xerial** [comment]
This problem happens since 1.0.1. sbg-pgp-1.0.0 has no problem.

**Comment by larsrh** [comment]
Appears to be fixed with sbt 1.0.0.
